ivadomed/loader/loader.py

```diff
@@ -358,27 +365,21 @@ def __getitem__(self, index):
         """
         seg_pair_slice, roi_pair_slice = self.indexes[index]
 
-        # Clean transforms params from previous transforms
-        # i.e. remove params from previous iterations so that the coming transforms are different
-        metadata_input = imed_loader_utils.clean_metadata(seg_pair_slice['input_metadata'])
-        metadata_roi = imed_loader_utils.clean_metadata(roi_pair_slice['gt_metadata'])
-        metadata_gt = imed_loader_utils.clean_metadata(seg_pair_slice['gt_metadata'])
```

Comment:
You removed it so that the preprocessing params (eg crop_params) are not wiped away. But it may involve that all the random transform will be the same at each `getitem` call since params already saved in metadata eg [here](https://github.com/neuropoly/ivado-medical-imaging/blob/master/ivadomed/transforms.py#L562). What do you think?
A solution would be to remove `crop_params` from this list, while keeping these lines of `clean_metadata`. I am happy to help with this.

Reply:
ahh yes I see! I only saw ROICrop (if crop_params in metadata...) , that's the first thing I tried -> removing crop_params from the list, but I was getting an error with ROICrop because the param was not updated here: https://github.com/neuropoly/ivado-medical-imaging/blob/master/ivadomed/loader/utils.py#L287 . If I remove crop_params but just from the update_metadata list I think everything should work